scripts/notify_team_new_comment.js

```diff
@@ -0,0 +1,85 @@
+const fs = require('fs');
+const path = require('path');
+
+function hasLabel(labels, name) {
+  return labels.some(label => label.toLowerCase() === name.toLowerCase());
+}
+
+module.exports = async ({ github, context, core }) => {
+  try {
+    const issueNumber = context.payload.issue.number;
+    const issueUrl = context.payload.issue.html_url;
+    const issueTitle = context.payload.issue.title;
+    const escapedTitle = issueTitle.replace(/"/g, '\\"');
+    const commentId = context.payload.comment.id;
+    const commentTime = new Date(context.payload.comment.created_at);
+    const commentAuthor = context.payload.comment.user.login;
+    const commentBody = context.payload.comment.body;
+    const repo = context.repo.repo;
+    const owner = context.repo.owner;
+    const slackWebhookUrl = process.env.SLACK_WEBHOOK_URL;
+    const communityWebhookUrl = process.env.SLACK_COMMUNITY_NOTIFICATIONS_WEBHOOK_URL;
+    const botMessage = `👋
+Thanks so much for your interest! This issue is currently reserved for the core team and isn’t available for assignment right now.
+If you’d like to get started contributing, please take a look at our [Contributing Guidelines](https://github.com/your-org/your-repo/blob/main/CONTRIBUTING.md) for tips on finding “help-wanted” issues, setting up your environment, and submitting a PR.
+We really appreciate your willingness to help — feel free to pick another issue labeled **help-wanted** and let us know if you have any questions. 😊`
+
+    const Close_Contributors = ['user1', 'user2'];
```

Comment:
Instead of writing it like this, it would be better to load it from a separate file. That way, it's easier to maintain and can also be reused in other places if needed.

Reply:
That'd be helpful, yes, thank you